pytrip/ctx.py

```diff
@@ -141,18 +140,6 @@ def create_dicom(self):
             data.append(_ds)
         return data
 
-    def write(self, path):
```

Comment:
Moved to cube class. It doesn't make sense to have multiple copies of the same method in each class.